common/models/role.js

```diff
@@ -97,7 +96,7 @@ module.exports = function(Role) {
       }
 
       roleModel.roleMappingModel.find({
-        where: { roleId: this.id, principalType: principalType },
+        where: {roleId: roleModel.roleMappingModel.id, principalType: principalType},
```

Comment:
I am afraid this is not a correct fix. I am landing https://github.com/strongloop/loopback/pull/2742 to fix this issue, please rebase on top of the new master after the change is merged.